### ISSUE TITLE
Process remains running after server close

### ISSUE THREAD

**Issue by simov** [curveball/core#160]
Any idea why after `server.close` the process remain running?

**Comment by evert** [comment]
Yes... any further thoughts about this? I don't know if this is easily fixable, but I suppose we could add a function to stop the interval.

**Comment by simov** [comment]
Definitely opting out will be awesome. Right now I'm monkey-patching setTimeout so that I can close it on server close ..
